src/main/java/com/deque/html/axecore/results/Platform.java

```diff
@@ -0,0 +1,50 @@
+package com.deque.html.axecore.results;
+
+public class Platform {
+    private String userAgent;
+    private String testMachine;
+
+    public String getUserAgent() {
+        return userAgent;
+    }
+
+    public void setUserAgent(String userAgent) {
+        this.userAgent = userAgent;
+    }
+
+    public String getTestMachine() {
+        return testMachine;
+    }
+
+    public void setTestMachine(String testMachine) {
+        this.testMachine = testMachine;
+    }
+
+
+
+    /**********************************
+     * Below is generated code.
+     * DO NOT MODIFY
+     * Regenerate instead
```

Comment:
How do we "regenerate" this?

Reply:
You'd have to ask the one who wrote attest-java-reporter, these files were copied from there. Not sure what IDE they used, but we can probably manually edit the equals, hashCode, and toString methods if we need to.